spec/fortnox/api/repositories/examples/all.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
-RSpec.shared_examples_for '.all' do |count|
+RSpec.shared_examples_for '.all' do
   describe '.all' do
     let(:response) do
       VCR.use_cassette("#{vcr_dir}/all") { subject.all }
     end
 
-    specify 'returns correct number of records' do
-      expect(response.size).to be count
+    it 'returns a list of records' do
+      expect(response).not_to be_empty
```

Comment:
Weaker test. Any reason why, other than not having to update the tests after each change? One possibility would be to put all of these in some kind of central config for specs instead and read them from there. That way you could update them all in the same place.

Reply:
Yes, that was the point. But you are correct, this is a weaker test. I'll add back the numbers.

I think I'll stick with the numbers in the specs. You need to update other numbers there as well, so better have all of them in that context.